test/conformance/testdata/pending_eligible_endpoints.yaml

```diff
@@ -2,3 +2,29 @@
 - patchCoreV1NamespacedPodResize
 - readCoreV1NamespacedPodResize
 - replaceCoreV1NamespacedPodResize
+- createResourceV1DeviceClass
```

Comment:
I'm still uncertain about adding endpoints to `pending_eligible_endpoints.yaml` which should be tested. Do we have a definition of the semantic of this file?

Is it "cannot be required by conformance yet" (for example, because the feature gate isn't GA yet) or is is "we should have tests for these but decided to postpone adding them"?